### ISSUE TITLE
No such file or directory: 'dxtbx.python'

### ISSUE THREAD

**Issue by Shahnawazgrewal** [nksauter/fv5080#3]
I have installed dials following the official [guidelines](https://dials.github.io/installation.html), afterwards i tired Install the image format class with `dxtbx.install_format -u https://raw.githubusercontent.com/nksauter/fv5080/master/FormatHDF5ImageDictionary.py`

I am receiving the following error:

```
Downloaded https://raw.githubusercontent.com/nksauter/fv5080/master/FormatHDF5ImageDictionary.py to /home/nawazsha/.dxtbx/FormatHDF5ImageDictionary.py
  found FormatHDF5ImageDictionary based on ['FormatHDF5', 'FormatStill']
Traceback (most recent call last):
  File "/home/nawazsha/dials/dials-dev20210909/build/../conda_base/bin/dxtbx.install_format", line 33, in <module>
    sys.exit(load_entry_point('dxtbx', 'console_scripts', 'dxtbx.install_format')())
  File "/home/nawazsha/dials/dials-dev20210909/modules/dxtbx/src/dxtbx/command_line/install_format.py", line 157, in run
    install_package(home_location, format_classes)
  File "/home/nawazsha/dials/dials-dev20210909/modules/dxtbx/src/dxtbx/command_line/install_format.py", line 75, in install_package
    procrunner.run(
  File "/home/nawazsha/dials/dials-dev20210909/conda_base/lib/python3.9/site-packages/procrunner/__init__.py", line 428, in wrapper
    return f(*args, **kwargs)
  File "/home/nawazsha/dials/dials-dev20210909/conda_base/lib/python3.9/site-packages/procrunner/__init__.py", line 524, in run
    p = subprocess.Popen(
  File "/home/nawazsha/dials/dials-dev20210909/conda_base/lib/python3.9/subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/home/nawazsha/dials/dials-dev20210909/conda_base/lib/python3.9/subprocess.py", line 1821, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'dxtbx.python'
```

Any lead to resolve it?

**Comment by phyy-nx** [comment]
Hello @Shahnawazgrewal, this issue is the same as was reported in cctbx/dxtbx#433, and is now fixed in cctbx/dxtbx#434.  If you update your modules/dxtbx repo using git pull, and then try `dxtbx.install_format` again, it should work.

If it works for you, let us know so we can close the issue?

**Comment by Shahnawazgrewal** [comment]
It is resolved, thank you for the pointers!
